src/entrypoints/my-redirect.ts

```diff
@@ -46,7 +46,9 @@ const createRedirectParams = (): string => {
   return `?${createSearchParam(userParams)}`;
 };
 
-const render = () => {
+let changingInstance = false;
```

Comment:
We should also set this to true if we redirect because there is no instance configured